### DISCUSSION TITLE
Handrit.is - searching the written past

### DISCUSSION THREAD

**Discussion by GudrunLaufey26** [network-digital-old-norse-studies/network-digital-old-norse-studies.github.io#13] [General]
Hi, Here is the abstract regarding the talk on Friday, Feb. 24th. 
24. February 2024.
Handrit.is - searching the written past.
A talk about the searching possibilities on Handrit.is. Handrit.is is a standardized database, all information are machine-readable and accessable through Git-Hub and openly acccessible with API‘s and can therefore be reused in different ways and projects. The database offers three searching fields: Browse, Power search and an open search string on the main page. The main emphasis of the talk was the different function of the various search possibilities and the challenges we are facing in the data registration. The balance between too much or too little information in the database and how we can give the best search results based on the data inserted. In the big picture it all depends on the registration. How accurate it is and according to the Tei-P5 guidelines, as a standardized and precise registration is the key to a good searching outcome, so the database can be used to its fullest.

**Comment by GudrunLaufey26** [comment]
Hi and thank you all for the last meeting, 
I didn´t know were to add this on this site, so I just placed it here for now. 
Kær kveðja
Guðrún Laufey

**Comment by BalduinLandolt** [comment]
@GudrunLaufey26 sorry I didn't get a notification when you posted this, so I've only just seen this now. Will add the page shortly. Thank you for the text!

**Comment by BalduinLandolt** [comment]
Merged and deployed just now
